package.json

```diff
@@ -33,16 +33,23 @@
     "@patternfly/react-table": "4.71.16",
     "@rhoas/app-services-ui-components": "1.38.2",
     "@rhoas/app-services-ui-shared": "0.16.0",
+    "ajv": "8.8.2",
+    "ajv-errors": "^3.0.0",
     "keycloak-js": "9.0.0",
+    "lodash.clonedeep": "^4.5.0",
     "lodash.isequal": "^4.5.0",
     "lodash.isequalwith": "^4.4.0",
+    "lodash.omit": "^4.5.0",
     "monaco-editor": "0.21.3",
     "monaco-editor-webpack-plugin": "2.1.0",
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "react-i18next": "11.16.7",
     "react-monaco-editor": "0.41.2",
-    "react-router-dom": "5.2.1"
+    "react-router-dom": "5.2.1",
+    "uniforms": "3.8.1",
+    "uniforms-bridge-json-schema": "3.8.1",
+    "uniforms-patternfly": "4.7.4"
```

Comment:
A recap.
- uniforms: dynamic form generation
- uniforms-bridge-json-schema: use json-schema with uniforms
- uniforms-patternfly: use PF with uniforms
- ajv: validate forms using json-schema rules
- ajv-errors: enhance error reporting of json-schema